components/button/button.stories.js

```diff
@@ -118,12 +118,6 @@ export const argTypesData = {
     },
   },
 
-  click: {
-    description: 'Native button click event',
-    table: {
-      type: { summary: 'event' },
-    },
-  },
```

Comment:
Why is this being removed here and not in Vue 2 version?

Reply:
In the Vue 2 version we have some native events defined like this just for documentation. In Vue 3 as it's defined in the emits we need to re-emit them without any additional action (in this case). So for that it's removed just in the Vue 3 implementation.